homeassistant/components/livisi/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+  "domain": "livisi",
+  "name": "LIVISI Smart Home",
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/livisi",
+  "requirements": ["aiolivisi==0.0.11"],
```

Comment:
Please don't pin the production requirements to exact versions.

https://github.com/StefanIacobLivisi/aiolivisi/blob/45e9c5fb4d15b3200c82e9f1fcf1fdad84e118b9/setup.py#L7

https://github.com/StefanIacobLivisi/aiolivisi/blob/45e9c5fb4d15b3200c82e9f1fcf1fdad84e118b9/requirements.txt

It's not recommended to use the requirements.txt file for the production requirements. It's better to separate dev requirements and production requirements. Dev requirements should normally be pinned to exact versions to be reproducable. For production, the library should be as forgiving as possible when specifying requirements to be as compatible as possible with other libraries that the application may install together with the library.

Reply:
I have only the "Heizkörperthermostat" in my setup, but in case you have some package I should test as a beta tester or something, just let me know. Note, I am quite new to HA, so I could only do simple installation steps, but I am willing to help, if useful.